tests/FakeItEasy.Tests.Approval.netstd/ApiApproval.cs

```diff
@@ -14,13 +15,14 @@ public class ApiApproval
         [Fact]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public void ApproveApiNetStd()
+        [SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods", MessageId = "System.Reflection.Assembly.LoadFrom", Justification = "By design")]
+        public void ApproveApi()

```

Comment:
renamed the method because FxCop was complaining about "Std"... I figured the suffix wasn't really necessary.